src/perf/bin/appmain.cpp

```diff
@@ -295,7 +290,7 @@ QuicKernelMain(
             OutBuffer,
             OutBufferSize,
             &OutBufferWritten,
-            240000);
+            INFINITE);
```

Comment:
is this safe? Seems like a fun way to have a new hang

Reply:
Yes, this is safe to do. This is basically the wait for test to finish call. Which in server mode can literally be forever, as we just want it to stay running. Cancelling with ctrl c, the shutdown backdoor, and a client test actually finishing will cause this to return.